contracts/RandomAuRa.sol

```diff
@@ -201,7 +210,8 @@ contract RandomAuRa is RandomBase, IRandomAuRa {
     bytes32 internal constant ALLOW_PUBLISH_SECRET = keccak256("allowPublishSecret");
     bytes32 internal constant CURRENT_SECRET = keccak256("currentSecret");
     bytes32 internal constant OWNER = keccak256("owner");
-
+    bytes32 internal constant COMMIT_PHASE_LENGTH = keccak256("commitPhaseLength"); // blocks
+    bytes32 internal constant COLLECT_ROUND_LENGTH = keccak256("collectRoundLength"); // blocks
```

Comment:
Please remove the `blocks` comment from here. These constants are just for using eternal storage upgradability.